src/metpy/calc/thermo.py

```diff
@@ -1556,14 +1556,66 @@ def saturation_vapor_pressure(temperature):
     Instead of temperature, dewpoint may be used in order to calculate
     the actual (ambient) water vapor (partial) pressure.
 
-    The formula used is that from [Bolton1980]_ for T in degrees Celsius:
+    Implemented solution from [Ambaum2020]_, Eq. 13,
+    .. math:: e = e_{s0} \frac{T_0}{T}^{(c_{pl} - c_{pv}) / R_v} \exp{
+    \frac{L_0}{R_v T_0} - \frac{L}{R_v T}}
 
-    .. math:: 6.112 e^\frac{17.67T}{T + 243.5}
+    """
+    latent_heat = water_latent_heat_vaporization._nounit(temperature)
+    heat_power = (mpconsts.nounit.Cp_l - mpconsts.nounit.Cp_v) / mpconsts.nounit.Rv
```

Comment:
After reading through section 5, I don't think we should rely on our constants here, but use the tuned value for $C_{pl} - C_{pv}$ that's used in the paper, 2180 J / kg / K (eq. 14). Our values are essentially assuming an ideal gas for water vapor, and nominally for the triple point; the discussion in Ambaum 2020 explains how their value minimizes errors against empirical data from IAPWS95 in the 0-100 degC range.

Reply:
I'm open to the idea that we allow users to override this value.

Reply:
Will attach a figure summary of constant differences here later to justify neglecting this.